src/pages/experiments/[accession_code]/[title]/index.js

```diff
@@ -0,0 +1,348 @@
+                      label={
+                        <TextHighlight>
+                          {experiment.alternate_accession_code}
+                        </TextHighlight>
+                      }
+                      href={getURLForAccessionCode(
+                        experiment.alternate_accession_code
+                      )}
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    />
+                  }
+                  textNull="None"
+                />
+              </InformationList>
+            </Box>
+          </FixedContainer>
+          <Box ref={tableRef}>
+            <FixedContainer>
+              <Box
+                elevation="medium"
+                pad={setResponsive('medium', 'large')}
+                margin={{ bottom: 'basex6' }}
+              >
+                <Row margin={{ bottom: 'medium' }}>
+                  <Column>
+                    <Heading
+                      level={2}
+                      margin={{ bottom: setResponsive('small', 'none') }}
+                    >
+                      Samples
+                    </Heading>
+                  </Column>
+                  <Column>
+                    <SamplesTableAction
+                      accessionCode={accessionCode}
+                      downloadableSamples={experiment.num_downloadable_samples}
+                    />
+                  </Column>
+                </Row>
+                {hasSamples ? (
+                  <SamplesTableManagerContextProvider>
+                    <SamplesTable
+                      allSamples={getFormattedExperiment(
+                        accessionCode,
+                        experiment.num_downloadable_samples
+                      )}
+                      sampleAccessionsInExperiment={{
+                        [experiment.accession_code]: experiment.samples.map(
+                          (sample) => sample.accession_code
+                        )
+                      }}
+                      queryToAdd={{
+                        experiment_accession_code: accessionCode
+                      }}
+                      sampleMetadataFields={experiment.sample_metadata}
+                      showOnlyAddedSamples
+                    />
+                  </SamplesTableManagerContextProvider>
+                ) : (
+                  <Spinner />
+                )}
+              </Box>
+            </FixedContainer>
+          </Box>
+        </Box>
+      </TextHighlightContextProvider>
+    </>
+  )
+}
+
+export const getServerSideProps = async ({ query }) => {
+  const response = await api.experiments.get(query.accession_code)
+
+  if (!response) {
+    return {
+      notFound: true
+    }
+  }
+
+  const platformNames = getPlatformNamesFromExperiment(response.samples)
+  const technology = getTechnologyFromExperiment(response.samples)
+  const experiment = { ...response, platform_names: platformNames, technology }
```

Comment:
We shouldn't be modifying the API response here. This is something that should be evaluated when we need it.